src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -174,15 +179,19 @@ private async Task ProcessRequestCoreAsync(StreamingMessage request)
             IOutputBindingsInfoProvider outputBindingsInfoProvider,
             IInputConversionFeatureProvider functionInputConversionFeatureProvider)
         {
+            var cts = new CancellationTokenSource();
```

Comment:
Are we always creating a `CancellationTokenSource` instance and adding to dictionary for every invocation. I wonder we should conditionally do this (check the function definition / Input bindings of the function being invoked, to determine Cancellation token is a parameter/input binding for this function)